docs/translations.rst

```diff
@@ -0,0 +1,310 @@
+      build_scripts     "build" scripts (copy and fixup #! line)
+      clean             clean up temporary files from 'build' command
+      install           install everything from build directory
+      install_lib       install all Python modules (extensions and pure Python)
+      install_headers   install C/C++ header files
+      install_scripts   install scripts (Python or otherwise)
+      install_data      install data files
+      sdist             create a source distribution (tarball, zip file, etc.)
+      register          register the distribution with the Python package index
+      bdist             create a built (binary) distribution
+      bdist_dumb        create a "dumb" built distribution
+      bdist_rpm         create an RPM distribution
+      bdist_wininst     create an executable installer for MS Windows
+      upload            upload binary package to PyPI
+      check             perform some checks on the package
+
+    Extra commands:
+      saveopts          save supplied options to setup.cfg or other config file
+      compile_catalog   compile message catalogs to binary MO files
+      develop           install package in 'development mode'
+      upload_docs       Upload documentation to PyPI
+      extract_messages  extract localizable strings from the project code
+      init_catalog      create a new catalog based on a POT file
+      test              run unit tests after in-place build
+      update_catalog    update message catalogs from a POT file
+      setopt            set an option in setup.cfg or another config file
+      install_egg_info  Install an .egg-info directory for the package
+      rotate            delete older distributions, keeping N newest files
+      bdist_wheel       create a wheel distribution
+      egg_info          create a distribution's .egg-info directory
+      alias             define a shortcut to invoke one or more commands
+      easy_install      Find/get/install Python packages
+      bdist_egg         create an "egg" distribution
+      dist_info         create a .dist-info directory
+      build_sphinx      Build Sphinx documentation
+
+    usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
+       or: setup.py --help [cmd1 cmd2 ...]
+       or: setup.py --help-commands
+       or: setup.py cmd --help
+
+.. seealso::
+
+    More details check out the following links:
+
+    - `Writing the Setup Configuration File <https://docs.python.org/3/distutils/configfile.html>`_.
+    - `How setup this file for babel configurations <http://babel.pocoo.org/en/latest/setup.html>`_.
+
+Extraction Configurations
+-------------------------
+
+First of all you have to get into the folder where you have your Sphinx theme and create a mapping file 
+called :file:`babel.cfg` that contains the **extraction from Jinja2 HTML templates** configurations. 
+For typical Sphinx extensions, this is what you want in there:
```

Comment:
@Blendify well thanks your feedback but I read the official reference and wrote this documentations because there are not a simple and really good reference about use Babel with Jinja2 engine template. For that I believe this minimal documentation useful for the guys that would like in the future to add more improvements about i18n on this package!